packages/react-composites/src/composites/CallComposite/components/buttons/Camera.tsx

```diff
@@ -15,6 +15,8 @@ export const Camera = (props: {
   styles?: ControlBarButtonStyles;
   splitButtonsForDeviceSelection?: boolean;
   disabled?: boolean;
+  /* @conditional-compile-remove(video-background-effects) */
+  onShowVideoEffectsOptions?: (showVideoEffectsOptions: boolean) => void;
```

Comment:
thinking on this I think that we should call the args here `showVideoEffectsPicker` just so there isn't confusion and this maybe being a options bag